packages/frontend/src/pages/insight-editor/insight-draft-container.tsx

```diff
@@ -88,6 +102,34 @@ export const InsightDraftContainer = ({ insight, draft, draftKey, onRefresh }) =
   const [upsertDraftResult, upsertDraft] = useMutation(DRAFT_UPSERT_MUTATION);
   const { error: upsertDraftError, fetching: isSavingDraft } = upsertDraftResult;
 
+  // UploadFile method for pasting images
+  const uploadFile = async (file, name) => {
+    // Upload file to IEX storage
+    const { data, error } = await urqlClient
+      .mutation<UploadSingleFileMutation>(UPLOAD_SINGLE_FILE_MUTATION, {
```

Comment:
There's a matching result type for `UploadSingleFileMutation`.  You can check the inferred type of the `uploadFile` function to find it.